s3psetup/s3psetup/cortx_s3_py_setup.py

```diff
@@ -0,0 +1,154 @@
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    if not error:
+      print ("Deleted s3recovery tool account successfully...")
+      return True 
+    print ("Delete s3recovery tool account failed with: {}".format(error))
+    return False
+
+  def accounts_cleanup(self, ldappasswd: str = None, s3background_cofig:str = "/opt/seagate/cortx/s3/s3backgrounddelete/config.yaml"):
+    """Clean up s3 accounts"""
+    if ldappasswd == None:
+      S3Cipher = CortxS3Cipher(None, True, 12, "openldap")
+      ldappasswd = S3Cipher.get_key()
+    rc1 = self.delete_background_delete_account(ldappasswd, 22, "s3backgroundaccesskey", s3background_cofig)
+    rc2 = self.delete_recovery_tool_account(ldappasswd, 22, "s3recoveryaccesskey", s3background_cofig)
+    return rc1 & rc2
+
+  def dependencies_cleanup(self):
+    """Clean up configs"""
+    log.debug("removing s3 dependencies")
+    cmd = "mv -f /opt/seagate/cortx/auth/resources/authserver.properties /opt/seagate/cortx/auth/resources/authserver.properties.bak"
+    output,error = subprocess.Popen(cmd, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    cmd = "mv -f /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml /opt/seagate/cortx/s3/s3backgrounddelete/config.yaml.bak"
+    output,error = subprocess.Popen(cmd, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    cmd = "mv -f /opt/seagate/cortx/auth/resources/keystore.properties /opt/seagate/cortx/auth/resources/keystore.properties.bak"
+    output,error = subprocess.Popen(cmd, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    cmd = "mv -f /opt/seagate/cortx/s3/conf/s3config.yaml /opt/seagate/cortx/s3/conf/s3config.yaml.bak"
+    output,error = subprocess.Popen(cmd, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    cmd = "mv -f /opt/seagate/cortx/s3/conf/s3_confstore.json /opt/seagate/cortx/s3/conf/s3_confstore.json.bak"
+    output,error = subprocess.Popen(cmd, universal_newlines=True, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
+    log.debug("\ncmd:{0},\noutput:{1},\nerror:{2}".format(cmd, output, error))
+    return True
+
+  def run(self):
+    parser = argparse.ArgumentParser(description='Cortx S3 Setup')
+    parser.add_argument("--cleanup", help='Cleanup S3 accounts and dependencies. Valid arg: both/accounts/dependencies')
```

Comment:
use "Valid values" instead of "Valid arg"

Also instead of using value as "both", use value "all"